File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {

        int k=0;
        int j=0;

        while(j < nums.length){
            int cur= nums[j];
            nums[k++]= cur;
            while(j < nums.length && nums[j] == cur ) j++;
        }
        return k;
    }
}

File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-0.cs
public class Solution {
    public int MinSubArrayLen(int target, int[] nums) {

        int st=0;
        int minLength=int.MaxValue;
        int sum=0;
        for(int end =0; end < nums.Length; end ++){   
            sum+= nums[end];

            while(sum >= target){
              minLength = Math.Min(minLength,end-st+1);
               sum -= nums[st];
               st++;
            }
        }

        return minLength == int.MaxValue ? 0 : minLength;
    }
}